tests/integration/test_dummy.py

```diff
@@ -1,2 +1,19 @@
-def test_dummy_unit() -> None:
-    assert 1 == 1  # noqa: S101
+import asyncio
+
+from adcm_aio_client.core.objects.cm import ClustersNode
+from adcm_aio_client.core.requesters import DefaultRequester
+from tests.integration.setup_environment import postgres_image_name, SetupManager, adcm_image_name
+
+
+async def test_paginated() -> None:
+    with SetupManager(postgres_image_name, adcm_image_name):
+        requester = DefaultRequester(base_url="http://127.0.0.1:8000", retries=3, retry_interval=60)
```

Comment:
can we detect base url of adcm with testcontainers tools?

Reply:
done